### ISSUE TITLE
QuickbloxWebRTC.framework missing x86_64 architecture

### ISSUE THREAD

**Issue by iliaskarim** [QuickBlox/quickblox-ios-sdk#77]
Trying to build the sample project in the latest xCode:

```
ld: warning: ignoring file ../Framework/QuickbloxWebRTC.framework/QuickbloxWebRTC, missing required architecture x86_64 in file ../Framework/QuickbloxWebRTC.framework/QuickbloxWebRTC (3 slices)
Undefined symbols for architecture x86_64:
  "_OBJC_CLASS_$_QBBackgroundTimer", referenced from:
      objc-class-ref in ConnectionManager.o
  "_OBJC_CLASS_$_QBICEServer", referenced from:
      objc-class-ref in StunViewController.o
  "_OBJC_CLASS_$_QBRTCClient", referenced from:
      objc-class-ref in SettingsCallViewController.o
      objc-class-ref in IncomingCallViewController.o
      objc-class-ref in CallViewController.o
      objc-class-ref in CallManager.o
  "_OBJC_CLASS_$_QBRTCConfig", referenced from:
      objc-class-ref in AppDelegate.o
      objc-class-ref in IncomingCallViewController.o
      objc-class-ref in CallViewController.o
      objc-class-ref in StunViewController.o
  "_OBJC_CLASS_$_QBSoundRouter", referenced from:
      objc-class-ref in CallViewController.o
      objc-class-ref in CallManager.o
  "_QuickbloxWebRTCFrameworkVersion", referenced from:
      -[LoginViewController version] in LoginViewController.o
  "_QuickbloxWebRTCRevision", referenced from:
      -[LoginViewController version] in LoginViewController.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

**Comment by soulfly** [comment]
http://quickblox.com/developers/Sample-webrtc-ios#System_requirements

iOS simulator 32 bit (doesn't support 64bit for now)

**Comment by NirmitDagly** [comment]
1. But how to solve this error for 32-bit simulator? means what are the necessary changes that I have to do to run the project?
2. By new guidelines of apple, I cannot upload app without supporting 64-bit architecture. So, I want to ask that Is Quickblox planning to release new framework for audio and video calling?

**Comment by soulfly** [comment]
QuickBlox WebRTC sdk support 64-bit device architecture
so you can build for 64bit and push it to the AppStore

it doesn't support 64bit for simulator only, so you can run only on 32bit simulator

**Comment by NirmitDagly** [comment]
![screen shot 2015-06-02 at 15 26 57](https://cloud.githubusercontent.com/assets/8401881/7933309/d6d2f1d4-093b-11e5-9489-47fe7373eeaf.png)
![screen shot 2015-06-02 at 15 27 31](https://cloud.githubusercontent.com/assets/8401881/7933323/ec2f46e0-093b-11e5-97b8-3fb6a08460f2.png)
I have attached screenshots of error and also I have attached second screenshot of my architecture configuration. Please advice me how to solve the errors.
Thanks.

**Comment by soulfly** [comment]
as I sad in previous messages you can't run it on 64-bit simulator

use 32-bit simulator

**Comment by Pro100Andrey** [comment]
Fixed in v 1.0.5
